tests/playwright/src/specs/podman-machine-rootless.spec.ts

```diff
@@ -0,0 +1,64 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import * as os from 'node:os';
+
+import { ResourceElementState } from '../model/core/states';
+import { CreateMachinePage } from '../model/pages/create-machine-page';
+import { ResourceConnectionCardPage } from '../model/pages/resource-connection-card-page';
+import { expect as playExpect, test } from '../utility/fixtures';
+import { deletePodmanMachine } from '../utility/operations';
+import { delay } from '../utility/wait';
+
+const PODMAN_MACHINE_NAME: string = 'podman-machine-rootless';
+const MACHINE_VISIBLE_NAME: string = 'Podman Machine rootless';
+
+test.beforeAll(async ({ pdRunner, welcomePage }) => {
+  pdRunner.setVideoAndTraceName('podman-rootless-machine-e2e');
+  process.env.KEEP_TRACES_ON_PASS = 'true';
+
+  await welcomePage.handleWelcomePage(true);
+});
+
+test.afterAll(async ({ pdRunner }) => {
+  await pdRunner.close();
+});
+
+test.describe('Rootless Podman machine Verification', () => {
+  test.skip(os.platform() === 'linux', 'runs only on windows and mac');
```

Comment:
This test.skip() should be outside the describe, because if it's outside it will be validated before the beforeAll, if it's inside the describe the beforeAll will still execute.